wandb/sdk/artifacts/artifact.py

```diff
@@ -1663,6 +1663,33 @@ def download(
         root: Optional[str] = None,
         recursive: bool = False,
         allow_missing_references: bool = False,
+    ) -> FilePathStr:
+        self._ensure_logged("download")
+
+        root = root or self._default_root()
+        self._add_download_root(root)
+        if wandb.run is None:
+            with wandb.init(
+                entity=self._source_entity,
+                project=self._source_project,
+                job_type="auto",
+                settings=wandb.Settings(silent="true"),
+            ) as run:
+                return FilePathStr(
```

Comment:
Nit: this is not needed, `run._download_artifact` already returns `FilePathStr`.